snap/local/start-mysqld.sh

```diff
@@ -6,10 +6,13 @@ if [ ! -f "${CONFIG_DIR}/initialized" ]; then
         echo "configuration folder ${CONFIG_DIR} does not exist."
         echo "copying default config to ${CONFIG_DIR}"
         cp -r $SNAP/etc/mysql $SNAP_COMMON
-        sed -i "s:/var/run/mysqld/mysqld.sock:$SNAP_COMMON/mysqld.sock:g" $SNAP_COMMON/mysql/mysql.conf.d/mysqld.cnf
-        sed -i "s:# sock:sock:g" $SNAP_COMMON/mysql/mysql.conf.d/mysqld.cnf
-        sed -i "s:/var/log/mysql/error.log:$SNAP_COMMON/error.log:g" $SNAP_COMMON/mysql/mysql.conf.d/mysqld.cnf
-        touch $CONFIG_DIR/initialized
+        chown -R snap_daemon:root $CONFIG_DIR
```

Comment:
would you mind adding some comments to this script (explaining the need for some of these command groups)? they would be super helpful for a new-comer to this repository, or to debug issues with the script down the road if something changes

Reply:
Comments added.